packages/fortinet_fortigate/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -285,27 +334,37 @@ processors:
             !ctx._temp.internal_interfaces.contains(ctx.observer.ingress.interface.name)
           )
         )
-  - remove:
-      field:
-        - _temp.time
-        - _temp
-        - syslog5424_sd
-        - fortinet.firewall.tz
-        - fortinet.firewall.date
-        - fortinet.firewall.devid
-        - fortinet.firewall.eventtime
-        - fortinet.firewall.time
-        - fortinet.firewall.duration
+  # Normalize the network direction
+  - script:
+      lang: painless
+      ignore_failure: true
+      params:
+        outgoing: outbound
+        incoming: inbound
+      source: >-
+        def k = ctx.network?.direction.toLowerCase();
```

```suggestion
        def k = ctx.network.direction.toLowerCase();
```
By the time we are here, we know `ctx.network` is not null.